modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/test/java/com/liferay/dynamic/data/mapping/form/web/internal/display/context/util/DDMFormGuestUploadFieldUtilTest.java

```diff
@@ -222,20 +270,22 @@ private void _mockDDMFormInstanceLocalService(
 		);
 	}
 
-	private DDMFormInstanceRecord _mockDDMFormInstanceRecord() {
+	private DDMFormInstanceRecord _mockDDMFormInstanceRecord(String ipAddress) {
 		DDMFormInstanceRecord ddmFormInstanceRecord = Mockito.mock(
 			DDMFormInstanceRecord.class);
 
 		Mockito.when(
 			ddmFormInstanceRecord.getIpAddress()
 		).thenReturn(
-			_IP_ADDRESS
+			ipAddress
 		);
 
 		return ddmFormInstanceRecord;
 	}
 
-	private HttpServletRequest _mockHttpServletRequest(boolean signedIn) {
+	private HttpServletRequest _mockHttpServletRequest(
+		boolean signedIn, String ipAddress) {
```

Comment:
@rijema rename it to `String remoteAddr` and sort alphabetically

Reply:
I also changed another method on the same class that I was using ipAddress to follow the same pattern. So, I changed to ipAddr.